pkg/apis/pipeline/v1beta1/task_types.go

```diff
@@ -24,17 +24,6 @@ import (
 	"knative.dev/pkg/kmeta"
 )
 
-const (
-	// TaskRunResultType default task run result value
-	TaskRunResultType ResultType = 1
-	// PipelineResourceResultType default pipeline result value
-	PipelineResourceResultType = 2
-	// InternalTektonResultType default internal tekton result value
-	InternalTektonResultType = 3
-	// UnknownResultType default unknown result type value
-	UnknownResultType = 10
-)
```

Comment:
This doesn't look right-- we're not getting rid of TaskRunResults, and I'm not sure what the other two result types are used for but I don't understand why we'd remove them.

Reply:
I have migrate it into the termination package and remove only the PipelineResourceResultType

Reply:
It is very confusing looking at PipelineResourceResult and meanwhile PipelineResourceResultType is a type of PipelineResourceResult 😵

Reply:
ah ok-- as a heads up results are no longer always written to termination messages (e.g. they can be written to sidecar logs) so pkg/termination might not be the best place. This const probably still belongs in pkg/apis.

Reply:
Hmmm yeah but I think here the `PipelineResourceResult` used to be called result but they are not really named as such. It felt to be different from the actual result_types. Instead `TaskRunResultType` and `InternalTektonResultType` and `PipelineResourceResultType` are introduced at   https://github.com/tektoncd/pipeline/issues/6197#issuecomment-1447031681

Reply:
Yes, I was going to suggest that. May be `pkg/results` or `pkg/internal/results`? I don;t think it should be tied to one way of extracting results.

Reply:
Also I wonder if this shall be in the apis since these are just go codes used internally in the code rather than what should be in the apis?